bin/lib/installable/script.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, install_context: InstallationContext, config: Dict[str, Any])
         super().__init__(install_context, config)
         self.install_path = self.config_get("dir")
         self.install_path_symlink = self.config_get("symlink", False)
-        self.fetch = self.config_get("fetch")
+        self.fetch = self.config_get("fetch", [])
```

Comment:
Was this change required?

Reply:
When I was testing, listing nightly compilers (and trying to install) would fail because there wasn't a fetch: for the dex2oat config. I had added the default empty list arg to avoid faking a fetch arg!